cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1460,6 +1460,26 @@
   <!-- Messaging System Configuration -->
 
   <property>
+    <name>messaging.cache.linger.ms</name>
+    <value>1000</value>
+    <description>
+      Time in milliseconds for an cached entry to stay in memory after it
+      is marked to be removed.
+    </description>
+  </property>
+
+  <property>
+    <name>messaging.cache.size.mb</name>
+    <value>30</value>
+    <description>
+      Memory in megabytes for the cache size used by the messaging service
+      for caching recently published messages. Currently only topics list
+      in the "messaging.system.topics" configuration will have caching enabled.
+      Set it to 0 to disable caching.
```

Comment:
Memory in megabytes for the cache size used by the messaging service
for caching recently published messages. Currently, only topics listed
in the ${messaging.system.topics} configuration have caching enabled.
Set it to 0 to disable caching.